### ISSUE TITLE
AbiWord 3.0.5-2 - Menu doesn't work on Haiku 32 bits

### ISSUE THREAD

**Issue by unspacyar** [haikuports/haikuports#7458]
When click on "File", "Edit", "View", etc.  menu, nothing happens.  The menu never is displayed.

I am using Haiku 32 bit (hrev56595)

**Comment by threedeyes** [comment]
What about other GTK apps on 32bit? Gimp, inkscape? This problem only in abiword?

**Comment by threedeyes** [comment]
I can't reproduce this bug. Menu works fine for me.

![VirtualBox_HaikuB2-32_21_11_2022_11_19_11](https://user-images.githubusercontent.com/2559260/202938836-22951468-4d28-4498-88e6-f8be69b80ce0.png)

**Comment by unspacyar** [comment]
Hi.  I noticed that I was using the **Be** ControlLook in the Appearence configuration.  I tried switch back to the **Default** one, and after that, AbiWord's menues are working fine.

I switched back to the **Be** ControlLook to confirm if there is some kind of compatibility  issues, but even now the AbiWord menu continue working fine.  Surely is something related to the ControlLook, but unfortunately I can not find a reproducible way to trigger it.
